test/testsuite.sh

```diff
@@ -28,15 +28,28 @@ echo "        TOMCAT_CYCLE_COUNT=$TOMCAT_CYCLE_COUNT"
 echo "        ITERATION_COUNT=$ITERATION_COUNT"
 echo "        IMG=$IMG"
 echo "        HTTPD_IMG=$HTTPD_IMG"
+if [ ! -z ${HTTPD_IMG+x} ]; then
```

Comment:
You meant `! -z ${HTTPD_IMG+x}` probably..?

Reply:
Oops I meant MPC_CONF

Reply:
Yeah, me too :D